src/d_iwad.c

```diff
@@ -142,22 +142,53 @@ static registry_value_t uninstall_values[] =
     },
 };
 
-// Value installed by the Collector's Edition when it is installed
+// Values installed by the GOG.com and Collector's Edition versions
 
-static registry_value_t collectors_edition_value =
+static registry_value_t gogcom_collectors_edition_values[] =
```

Comment:
The name seems confusing. Can we give it a more meaningful name. For example: root_path_registry_keys?